services/board/src/Domain/MovingStrategies/Moving.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace CliChess\Board\Domain\MovingStrategies;
+
+use CliChess\Board\Domain\Square;
+
+interface Moving
```

Comment:
I think this interface should be called `MovinStrategy` and all implementations should be suffixed with `MovinStrategy` as well.

- `DiagnalMovinStrategy`
- `LShapeMovinStrategy`
- ...